e2e/test/scenarios/admin/datamodel/datamodel.cy.spec.ts

```diff
@@ -1081,70 +986,66 @@ describe("scenarios > admin > databases > table", () => {
-        .within(() => {
-          cy.findByText("INTEGER").should("be.visible");
-          cy.findByPlaceholderText("Select a semantic type").should(
-            "have.value",
-            "Foreign Key",
-          );
-          cy.findByPlaceholderText("Select a target").should(
-            "have.value",
-            "People → ID",
-          );
-        });
+      clickField("User ID");
+      cy.findByLabelText("Data type")
+        .should("be.visible")
+        .and("have.value", "INTEGER");
+      cy.findByPlaceholderText("Select a semantic type").should(
+        "have.value",
+        "Foreign Key",
+      );
+      cy.findByPlaceholderText("Select a target").should(
+        "have.value",
+        "People → ID",
+      );
 
-      cy.findByTestId("column-TAX")
-        .scrollIntoView()
-        .within(() => {
-          cy.findByText("DOUBLE PRECISION").should("be.visible");
-          cy.findByPlaceholderText("Select a semantic type").should(
-            "have.value",
-            "No semantic type",
-          );
-        });
+      clickField("Tax");
+      cy.findByLabelText("Data type")
+        .should("be.visible")
+        .and("have.value", "DOUBLE PRECISION");
+      cy.findByPlaceholderText("Select a semantic type").should(
+        "have.value",
+        "No semantic type",
+      );
 
-      cy.findByTestId("column-DISCOUNT")
-        .scrollIntoView()
-        .within(() => {
-          cy.findByText("DOUBLE PRECISION").should("be.visible");
-          cy.findByPlaceholderText("Select a semantic type").should(
-            "have.value",
-            "Discount",
-          );
-        });
+      clickField("Discount");
+      cy.findByLabelText("Data type")
+        .should("be.visible")
+        .and("have.value", "DOUBLE PRECISION");
+      cy.findByPlaceholderText("Select a semantic type").should(
+        "have.value",
+        "Discount",
+      );
 
-      cy.findByTestId("column-CREATED_AT")
-        .scrollIntoView()
-        .within(() => {
-          cy.findByText("TIMESTAMP").should("be.visible");
-          cy.findByPlaceholderText("Select a semantic type").should(
-            "have.value",
-            "Creation timestamp",
-          );
-        });
+      clickField("Created At");
+      cy.findByLabelText("Data type")
+        .should("be.visible")
+        .and("have.value", "TIMESTAMP");
+      cy.findByPlaceholderText("Select a semantic type").should(
+        "have.value",
+        "Creation timestamp",
+      );
     });
   });
 
-  describe.skip("turning table visibility off shouldn't prevent editing related question (metabase#15947)", () => {
+  describe("turning table visibility off shouldn't prevent editing related question (metabase#15947)", () => {
     it("simple question (metabase#15947-1)", () => {
       turnTableVisibilityOff(ORDERS_ID);
       H.visitQuestion(ORDERS_QUESTION_ID);
-      H.filter();
+
+      H.queryBuilderHeader().findByText("View-only").should("be.visible");
```

Comment:
Unskipped and updated: #48791.
The other test `metabase#15947-2` is still skipped. We could probably removed it, but maybe good to keep for reference.